Example/Sources/AppDelegate.swift

```diff
@@ -30,9 +30,15 @@ final internal class AppDelegate: UIResponder, UIApplicationDelegate {
     var window: UIWindow?
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+
+        let masterVC = NavigationController(rootViewController: LaunchViewController())
```

Comment:
I believe example app uses full naming (aka `ViewController`, not `VC`), could we stick with that please?